toolkit/tools/internal/logger/log.go

```diff
@@ -179,8 +185,15 @@ func ReplaceStderrFormatter(newFormatter logrus.Formatter) (oldFormatter logrus.
 	return stderrHook.ReplaceFormatter(newFormatter)
 }
 
-func initStderrLogInternal(callerFilePath string) {
-	const useColors = true
+func initStderrLogInternal(callerFilePath string, color string) {
+	var useColors bool
+	if color == "auto" || color == "always" {
+		useColors = true
+	} else if color == "never" {
+		useColors = false
+	} else {
+		useColors = true
+	}
```

Comment:
Could we move `auto`, `always`, and `never` to well-named constants? This way they raw string will be defined in only one place and then referenced with the constant. That makes it harder to create a hard to spot bug with a typo and also any modifications to the level (although unlikely in this case) would be only necessary in one place in the code.

Reply:
They could also be a type, something like:
```go
type LoggerColourMode string
const (
	ColourModeAuto LoggerColourMode "auto"
	ColourModeAlways LoggerColourMode "always"
	ColourModeNever LoggerColourMode "never"
)
```